source/rest-v2.md

```diff
@@ -697,6 +976,21 @@ The JSON response returns a list of collections with the following attributes:
 | Turkish    | tr   |
 | Ukranian   | uk   |
 
+# Extraction Options
+
+The following fields defines options to extract hotspots from a PDF
+
+| Name                | Type   | Required | Description                                                                                                                                       |
+| ------------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| links               | Object | No       | Extracts link hotspots for any URLs, document links, and email addresses found in PDF file (see object definition table bellow)                   |
+| products            | Object | No       | Extracts product hotspots from any SKU found in PDF file, matching product feed or product library set (see object definition table bellow)       |
+
+Both 'links' and 'products' allow the fields bellow:
+
+| Name                | Type     | Required | Description                                                                                                                              |
+| ------------------- | -------- | -------- | -----------------------------------------------------------------------------------------------------------------------------------------|
+| icons               | Boolean  | No       | Show/hide hotspot icon indication, is not applicable hotspot extracted from pdf annotations(those are defined in the annotantion itself) |
```

Comment:
`Show/hide hotspot icon indication. It is not applicable if the hotspot is extracted from PDF annotations (those are defined in the annotation itself)`